proto/mixer.proto

```diff
@@ -526,6 +526,16 @@ message RelatedPlacesInfo {
   reserved 4, 5, 6;
 }
 
+message GetStatVarPathRequest {
```

Comment:
This is actually a path of SVGs, so should it be GetStatVarGroupPathRequest ?

Reply:
Would like to de-emphasize stat var group in API naming and URL as it's more of a way to represent a set of stat vars. From user perspective, they are interested in stat var.